### PR TITLE
Use docconvert to normalize docstring to Google style

### PR DESCRIPTION
**Pull request by justinchuby**
### Description

Use docconvert to normalize docstring to Google style and enable pydocstyle checks in ruff.

### Motivation and Context

Standardize docstring style.

### CODE REVIEW

File: onnx/backend/test/case/node/affinegrid.py [hunk 1]
```diff
@@ -1,209 +1,209 @@
...
+
+            test_name = "test_affine_grid_3d"
+            if align_corners == 1:
+                test_name += "_align_corners"
+            expect(
+                node,
+                inputs=[theta_3d, np.array([N, C, D, H, W], dtype=np.int64)],
+                outputs=[grid],
+                name=test_name,
+            )
```
Comment (justinchuby) [inline onnx/backend/test/case/node/affinegrid.py:209]: line breaks?

File: onnx/reference/ops/op_topk.py [hunk 2]
```diff
@@ -106,13 +102,12 @@ def _run(self, data, ink, axis=None):  # type: ignore
...
     def _run(self, data, ink, axis=None, largest=None, sorted=None):  # type: ignore
-        """
-        Runtime for operator *TopK*.
+        """Runtime for operator *TopK*.
         The implementation is not the most efficient
         as it sorts everything then extracts the top *k*
         values.
 
-        .. warning::
+        Warning:
```
Comment (justinchuby) [inline onnx/reference/ops/op_topk.py:110]: Revert

File: onnx/test/function_test.py
Comment (justinchuby) [inline onnx/test/function_test.py:1]: Revert

File: onnx/test/compose_test.py
Comment (justinchuby) [inline onnx/test/compose_test.py:1]: Revert

File: onnx/reference/reference_evaluator.py [hunk 3]
```diff
@@ -21,36 +21,36 @@
 
 
 class ReferenceEvaluator:
-    """
-    Computes the outputs of an ONNX proto
+    """Computes the outputs of an ONNX proto
```
Comment (justinchuby) [inline onnx/reference/reference_evaluator.py:24]: Update

File: onnx/reference/ops/op_scatter_elements.py [hunk 4]
```diff
@@ -9,13 +9,12 @@
...
 
 def scatter_elements(data, indices, updates, axis=0, reduction=None):  # type: ignore
-    """
-    ::
-        // for 3-dim and axis=0
-        //    output[indices[i][j][k]][j][k] = updates[i][j][k]
-        // for axis 1
-        //    output[i][indices[i][j][k]][k] = updates[i][j][k]
-        // and so on
+    """::
```
Comment (justinchuby) [inline onnx/reference/ops/op_scatter_elements.py:12]: Revert

File: onnx/reference/ops/op_hamming_window.py [hunk 5]
```diff
@@ -9,8 +9,7 @@
 
 
 class HammingWindow(_CommonWindow):
-    """
-    Returns
+    """Returns
```
Comment (justinchuby) [inline onnx/reference/ops/op_hamming_window.py:12]: Update

File: onnx/reference/ops/op_blackman_window.py [hunk 6]
```diff
@@ -10,8 +10,7 @@
 
 
 class BlackmanWindow(_CommonWindow):
-    """
-    Returns
+    """Returns
```
Comment (justinchuby) [inline onnx/reference/ops/op_blackman_window.py:13]: Update

**Review by gramalingam — APPROVED** [review]

**Pull request merged**